src/libraries/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.Async.cs

```diff
@@ -37,6 +37,24 @@ public async Task<Stream> OpenAsync(CancellationToken cancellationToken = defaul
         }
     }
 
+    public async Task<Stream> OpenAsync(string password, CancellationToken cancellationToken = default)
```

Comment:
This duplicates the existing OpenAsync method, you can you now redirect the old method to call this one with empty password?